run.py

```diff
@@ -106,6 +121,39 @@ def config_validator():
         app.config['TESTING'] = True
         print("Starting server in ~*TEST MODE*~")
         app.run(host='localhost')
+    elif args.backup:
+        # To verify if issues.db exist, simply calling os.path.exists is not enough.
+        # Both os.path.exists and os.path.isfile returns true even when file is not there in the path
+        # Every time the script is re run, os.path.exists returns true because of cache.
+        # To avoid this, we open the file in try catch - just to be sure
+        if os.path.isfile(os.path.join(os.getcwd(), 'issues.db')):
+            try:
+                with open(os.path.join(os.getcwd(), 'issues.db')) as file:
+                    if not os.path.exists(app.config['BACKUP_DEFAULT_DEST']):
+                        print 'backup db folder does not exist'
+                        os.makedirs(app.config['BACKUP_DEFAULT_DEST'])
+                    else:
+                        print 'backup db folder exists'
+                        # Retain last 3 recent backup files
+                        no_of_backup_files = len(os.listdir(app.config['BACKUP_DEFAULT_DEST']))
+                        if no_of_backup_files > 3:
+                            for idx, val in enumerate(os.listdir(app.config['BACKUP_DEFAULT_DEST'])):
+                                if idx < no_of_backup_files - 2:
+                                    os.remove(app.config['BACKUP_DEFAULT_DEST'] + val)
+                    current_ts = time.time()
+                    time_stamp = datetime.datetime.fromtimestamp(current_ts).strftime('%Y-%m-%d-%H:%M:%S')
+                    issue_backup_db = 'issues_backup_' + time_stamp + '.db'
+                    os.rename(os.getcwd() + '/issues.db', app.config['BACKUP_DEFAULT_DEST'] + issue_backup_db)
+                    try:
+                        os.remove(os.path.join(os.getcwd(), 'issues.db'))
+                    except OSError:
+                        print 'Backup issues.db complete'
```

Comment:
@miketaylr  I get a weird issue here. Even though I try multiple times to verify that issues.db is not available in /Users/username/webcompat.com, I get true for line number 129
`if os.path.isfile(os.path.join(os.getcwd(), 'issues.db')):`
 when I rerun the script.

Even in try[line 131]:
` with open(os.path.join(os.getcwd(), 'issues.db')) as file:`
I have placed an additional check to open issues.db which passes. 

I am not able to figure out why this happens.